tests_e2e/test_suites/ext_update.yml

```diff
@@ -0,0 +1,8 @@
+#
+#    This test verifies that the agent reports correct status and code when operation fails during the ext update.
+#
+name: "ExtensionUpdate"
+tests:
+  - "ext_update/ext_update_failure.py"
+images: "random(endorsed)"
+owns_vm: true
```

Comment:
is it necessary for this test to own the vm?

Could we add cleanup at the end of the test to uninstall the failed ext

Reply:
Yes, its needed because other scenarios might install CSE already in the vm, then this scenario won;t work

Reply:
ok, in other scenarios we don't do it. If vm runs other scenarios first, we will be a trouble. I'll add cleanup but still required own_vm:true

Reply:
I was thinking about cleanup after test run. We are going down that route of cleanup other scenario mess.

Reply:
I just realized we can't let this vm shared by others as it may create issues for other scenarios if they use CSE. Once we modified the handlermanfiest, we need to reset it to uninstall extension. For that I need to too much logic for simple scenario. Given we run in single vm, I wouldn't worry about creating new environment.